index.html

```diff
@@ -12,12 +12,23 @@
   gtag('config', 'AW-376259107');
   gtag('config', 'G-G55HQ8G49M');
   gtag('config', 'UA-219916432-1');
+
+  // Adds `transaction_id` query parameter to a link if one is specified in the
+  // "Transaction ID" input field.
+  function addTransactionId(linkElement){
+    const transactionId = document.getElementById("conversion-transaction-id-input").value;
+    if (transactionId != '') {
+      linkElement.href += "?transaction_id=" + transactionId;
+    }
+  }
 </script>
 
 </head>
 
 <title>Mirror Test</title>
 <p> Mirror Test </p>
-<a href = "conversion2.html">Conversion2</a>
-<a href = "ga-conversion.html">GA Conversion</a>
-<a href = "legacy-ga-conversion.html">Legacy-GA Conversion</a>
+<label for="conversion-transaction-id-input"> Transaction ID: </label>
+<input type="number" id="conversion-transaction-id-input" name="transaction-id"/>
```

Comment:
nit: I wonder if type="text" is better here?

Reply:
I copied this from https://github.com/mirrortesting/mirrortesting.github.io/blob/main/index.html#L27.

But it seems like transaction ids can contain alphabetical and special characters (https://support.google.com/google-ads/answer/6386790?hl=en), so text probably makes more sense here.